Library/Homebrew/utils/analytics.rb

```diff
@@ -69,12 +77,57 @@ def report(type, metadata = {})
         nil
       end
 
-      def report_event(category, action, label = os_arch_prefix_ci, value = nil)
-        report(:event,
-               ec: category,
-               ea: action,
-               el: label,
-               ev: value)
+      def report_influx(category, action, on_request, info = {})
+        return unless ENV["HOMEBREW_ANALYTICS_ENABLE_INFLUX"]
+
+        # Append general information to device information
+        tags = info.merge(action: action, on_request: on_request ? 1 : 0)
+                   .compact_blank
+                   .map { |k, v| "#{k}=#{v.to_s.sub(" ", "\\ ")}" }
```

Comment:
```suggestion
                   # convert to key/value parameters
                   .map { |k, v| "#{k}=#{v.to_s.sub(" ", "\\ ")}" }
```

do we need to do any other URL escaping/encoding here?

Reply:
No, just need to escape whitespace since it's the separator between different values.

Reply:
cool 👍🏻